packages/erxes-ui/src/components/richTextEditor/extensions/Mention.ts

```diff
@@ -1,26 +1,26 @@
-import Mention from '@tiptap/extension-mention';
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Consider using consistent naming for imports to improve readability.